scripts/sv/scanBam.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -eu
+
+if [[ -z ${GATK_DIR+x} || -z ${CLUSTER_NAME+x} || -z ${MASTER_NODE+x} || -z ${PROJECT_OUTPUT_DIR+x} ]]; then 
+    if [[ "$#" -ne 3 ]]; then
+        echo "Please provide: local directory of GATK build, cluster name, output dir on the cluster"
+    exit 1
+    fi
+    GATK_DIR=$1
+    CLUSTER_NAME=$2
+    OUTPUT_DIR=$3
+    MASTER_NODE="hdfs://""$CLUSTER_NAME""-m:8020"
+    PROJECT_OUTPUT_DIR="$MASTER_NODE"/"$OUTPUT_DIR"
+fi
+
+cd "$GATK_DIR" 
+
+./gatk-launch FindBreakpointEvidenceSpark \
+    -R "$MASTER_NODE"/reference/Homo_sapiens_assembly19.fasta \
+    -I "$MASTER_NODE"/data/NA12878_PCR-_30X.bam \
```

Comment:
It might be nice to parameterize the bam name out of this script so it could be reused on other hg19 data.